### PR TITLE
EWL-4429 Fixes typography for Promo as inline for News Article

### PR DESCRIPTION
**Pull request by designerneil**
Margin around the block

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**

- [EWL-4429: Fix typography and padding](https://issues.ama-assn.org/browse/EWL-4429)

## Description

```
Nick Aleck added a comment - 3 days ago
Topic Promo –

body font should be 400 weight
there needs to be increased padding around the promo (Viewing in Chrome)
```

Addresses the margin and typography weight

## To Test

- [ ] http://localhost:3000/?p=molecules-inline-block-copy 
- [ ] Observe font weight
- [ ] http://localhost:3000/?p=pages-news
- [ ] Observe margin around the inline block content

## Visual Regressions

none

## Relevant Screenshots/GIFs

screen shot 2018-01-26 at 2 10 44 pm

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

### CODE REVIEW

File: styleguide/source/assets/scss/00-base/_font-variables.scss [hunk 1]
```diff
@@ -3,9 +3,9 @@ $myriad-pro: "myriad-pro", "Helvetica", "Arial", "Open Sans", sans-serif;
 $kepler-std: "kepler-std", Georgia, Times, "Times New Roman", serif;
 
 // Font weight variables
-$font-weight-regular: 100;
-$font-weight-semibold: 400;
-$font-weight-bold: 600;
+$font-weight-regular: 400;
+$font-weight-semibold: 600;
+$font-weight-bold: 900;
```
Comment (labbydev) [inline styleguide/source/assets/scss/00-base/_font-variables.scss]: It looks like this also updated the font weights on the article stubs in the column teaser and the four up teaser, were those also intended to be updated?

**Review by labbydev — CHANGES_REQUESTED** [review]
@designerneil can you also update it so that the promo as inline is full width on mobile in order to match the wireframe?

File: styleguide/source/assets/scss/00-base/_font-variables.scss (hunk 1 above)
Comment (designerneil) [reply styleguide/source/assets/scss/00-base/_font-variables.scss]: yes. it seems like UX wanted regular type to be 400 and semibold 600. 100 font-weight was too light.

**Comment by labbydev** [comment]
@designerneil can you also update it so that the promo as inline is full width on mobile in order to match the wireframe?

**Comment by designerneil** [comment]
@labbydev where is mobile wireframe?

**Comment by labbydev** [comment]
@designerneil https://sketch.cloud/s/L0dLQ/p/news-article/article-mobile is the mobile wireframe. Using the arrows at the top where it says `2/3` you can switch between Mobile, Tablet and Desktop wireframes.

**Comment by designerneil** [comment]
adds responsive breakpoints to enable the page to go full width when in mobile view.

**Comment by labbydev** [comment]
@designerneil this is appearing as full-width always now.

screen shot 2018-01-31 at 11 13 58 am

It also looks like the markup may be a bit off

```
<div class="ama__inline-block-copy">
  <div class="ama__rule"></div>
  <div class="ama__inline-block-copy--content">
    <h2 class="ama__h2">Title Text</h2>
    <p class=""> A paragraph (from the Greek paragraphos, "to write beside" or "written beside") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose.</p>
    
  <a href="#" class="ama__button  ama__button--secondary" title="Sign Up">
      Sign up
  </a>
  </div>
  <div class="ama__rule"></div>
</div>
```

It looks like you put the breakpoints for `&___copy` which the class should probably just be `ama__inline-block-copy`. Also `ama__inline-block-copy--content` should be a sub-element and therefore should be `ama__inline-block-copy__content`.

**Comment by designerneil** [comment]
Thanks @labbydev for your suggestions I've simplified the inline-block css to use ama__inline-block-copy--content as oppose to &__copy which didn't do anything.

File: styleguide/source/_patterns/02-molecules/inline-block-copy.twig [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {% set heading, paragraph, button = inlineBlockCopy.heading, inlineBlockCopy.paragraph, inlineBlockCopy.button %}
 
-<div class="ama__inline-block-copy">
+<div class="ama__inline-block-copy col-sm-12 col-md-12 col-lg-4">
```
Comment (labbydev) [inline styleguide/source/_patterns/02-molecules/inline-block-copy.twig]: `col-md-12` and `col-sm-12` are still setting this element to `width: 100%` and it doesn't appear that `col-lg-4` is having any effect at all. I would recommend handling the sizing directly within the `scss` for `ama__inline-block-copy`

**Review by labbydev — CHANGES_REQUESTED** [review]

File: styleguide/source/_patterns/02-molecules/inline-block-copy.twig (hunk 2 above)
Comment (designerneil) [reply styleguide/source/_patterns/02-molecules/inline-block-copy.twig]: @labbydev perhaps I'm confused how this should be viewed on mobile devices but I thought we wanted the inline block to render at 100% on tablet and mobile and around 25% on desktop. If the grid sizing is not working then we will need to revisit it right away.

**Comment by designerneil** [comment]
## **Mobile**
screen shot 2018-01-31 at 4 10 32 pm

## **Tablet**
screen shot 2018-01-31 at 4 10 24 pm

## **Desktop**
screen shot 2018-01-31 at 4 10 15 pm

**Comment by designerneil** [comment]
@labbydev the inline block will remain at 25% until the viewport reaches mobile size. Thanks again

**Comment by labbydev** [comment]
@designerneil now the inline promo is working properly but the sidebar is broken now.

localhost_3000_patterns_05-pages-news_05-pages-news html

**Comment by designerneil** [comment]
@labbydev sorry about the column thing. I was introduced right before I pushed my branch when I was rebasing it off of develop.

**Pull request closed without merging**